src/TerraformImportPlugin.groovy

```diff
@@ -0,0 +1,49 @@
+import static TerraformEnvironmentStage.PLAN_COMMAND
+
+class TerraformImportPlugin implements TerraformEnvironmentStagePlugin, TerraformImportCommandPlugin {
+
+    public static void init() {
+        TerraformImportPlugin plugin = new TerraformImportPlugin()
+
+        BuildWithParametersPlugin.withStringParameter([
+            name: "IMPORT_RESOURCE",
+            description: "Run `terraform import` on the resource specified prior to planning and applying."
+        ])
+        BuildWithParametersPlugin.withStringParameter([
+            name: "IMPORT_TARGET_PATH",
+            description: "The path in the Terraform state to import the spcified resource to."
+        ])
```

Comment:
Would this work when there is more than one environment?  I thought BuildWithParametersPlugin prompts the user at the start of the pipeline.  If I have QA/UAT/Prod environments in my pipeline, won't it be almost guaranteed that the resource ARNs will be different per environment?  Wouldn't the user need to be prompted on a per-environment basis?

Reply:
That is a problem, I think. I'll see what can be done to mitigate that...

Reply:
It looks like you addressed this problem by providing a third `IMPORT_ENVIRONMENT` parameter.  Is that correct?

Reply:
That's correct - it's the simplest MVP I came up with. It's not my favorite, but it does work to fix the problem.